src/app/course-list/models/course-item.ts

```diff
@@ -2,28 +2,17 @@ import {CourseItemInterface} from './course-item.model';
 
 // TODO: generate id and set creationDate in condtructor
 export class CourseItem implements CourseItemInterface {
```

Comment:
There is a convention for interface naming. **I<interface_name>**
So it's better to rename it in `ICourseItem`